contracts/test/LightClient.t.sol

```diff
@@ -369,45 +353,41 @@ contract LightClient_newFinalizedState_Test is LightClientCommonTest {
             // check if LightClient.sol states are updated correctly
             assertEq(abi.encode(lc.getFinalizedState()), abi.encode(states[i]));
             // check against hardcoded epoch advancement expectation
-            if (i == BLOCKS_PER_EPOCH_TEST) {
-                // first block of a new epoch (from epoch 2) should update the following
-                assertEq(lc.currentEpoch(), 2);
-                bytes32 genesisComm = lc.computeStakeTableComm(genesis);
-                LC.LightClientState memory lastBlockInFirstEpoch = states[i - 1];
-                bytes32 firstEpochComm = lc.computeStakeTableComm(lastBlockInFirstEpoch);
-                assertEq(lc.votingStakeTableCommitment(), genesisComm);
-                assertEq(lc.frozenStakeTableCommitment(), firstEpochComm);
-                assertEq(lc.votingThreshold(), genesis.threshold);
-                assertEq(lc.frozenThreshold(), lastBlockInFirstEpoch.threshold);
-            } else {
-                assertEq(lc.currentEpoch(), 1);
-                bytes32 stakeTableComm = lc.computeStakeTableComm(genesis);
-                assertEq(lc.votingStakeTableCommitment(), stakeTableComm);
-                assertEq(lc.frozenStakeTableCommitment(), stakeTableComm);
-                assertEq(lc.votingThreshold(), genesis.threshold);
-                assertEq(lc.frozenThreshold(), genesis.threshold);
-            }
+            //            if (i == statesLen - 1) {
+            //                // first block of a new epoch (from epoch 2) should update the
+            // following
+            //                bytes32 genesisComm = lc.computeStakeTableComm(genesis);
+            //                LC.LightClientState memory lastBlockInFirstEpoch = states[i - 1];
+            //                bytes32 firstEpochComm =
+            // lc.computeStakeTableComm(lastBlockInFirstEpoch);
+            //                assertEq(lc.votingStakeTableCommitment(), genesisComm);
+            //                assertEq(lc.frozenStakeTableCommitment(), firstEpochComm);
+            //                assertEq(lc.votingThreshold(), genesis.threshold);
+            //                assertEq(lc.frozenThreshold(), lastBlockInFirstEpoch.threshold);
```

Comment:
let's just safely remove these, can always add back or find these through git history. let's keep the testing logic clean.